src/controllers/presentation.ts

```diff
@@ -199,6 +202,7 @@ export class PresentationController {
 		const cheqdPresentation = new CheqdW3CVerifiablePresentation(presentation);
 
 		if (makeFeePayment) {
+			// ToDo: Add Payments tracker
```

Comment:
Since this is in scope of this PR, let's implement + include in Credential Status operations as well.

Reply:
Hey! Such payment tracking feature has been implementing in the PR with minor tracking things.